src/main/java/core/basesyntax/figures/Triangle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.colors.Color;
+
+public class Triangle extends RightTriangle {
```

Comment:
let's remove this class for now

if we decide to have it, we have to refactor code: public class RightTriangle extends Triangle - this would be more logical